src/btf.cpp

```diff
@@ -0,0 +1,257 @@
+#include <iostream>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/utsname.h>
+#include <string.h>
+#include <linux/limits.h>
+#include "btf.h"
+#include "types.h"
+#include "bpftrace.h"
+
+#ifdef HAVE_LIBBPF_BTF_DUMP
+#include <linux/bpf.h>
+#include <linux/btf.h>
+#include <bpf/btf.h>
+#include <bpf/libbpf.h>
+
+namespace bpftrace {
+
+static unsigned char *get_data(const char *file, ssize_t *sizep)
+{
+  struct stat st;
+
+  if (stat(file, &st))
+    return NULL;
```

Comment:
And also for the rest of this file
```suggestion
    return nullptr;
```

Reply:
ok